etl/steps/data/garden/un/2024-07-25/refugee_data.meta.yml

```diff
@@ -0,0 +1,279 @@
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Migration
+        - War & Peace
+    unit: people
+    short_unit: ""
+    processing_level: major
+    description_processing: |-
+      - The data collected by the UNHCR describes movement of refugees between individual regions. To calculate the total number of refugees in a region, we sum up the data from all origin countries. The same applies to calculating all refugees from one country of origin.
+      - To calculate the number of refugees per 1,000 people in a country, we divide the number of refugees by the total population of the country and multiply by 1,000. We use our long term population data for this calculation.
+    display:
+      numDecimalPlaces: 0
+  description_international_protection: Other people in need of international protection refers to people who are outside their country or territory of origin, typically because they have been forcibly displaced across international borders, who have not been reported under other categories (asylum-seekers, refugees, people in refugee-like situations) but who likely need international protection, including protection against forced return, as well as access to basic services on a temporary or longer-term basis.
+  description_interally_displaced: Internally displaced persons (IDPs) are persons or groups of persons who have been forced or obliged to flee or to leave their homes or places of habitual residence, in particular as a result of, or in order to avoid the effects of armed conflict, situations of generalized violence, violations of human rights or natural or human-made disasters, and who have not crossed an internationally recognized State border. For the purposes of UNHCR’s statistics, this population includes only conflict-generated IDPs to whom the Office extends protection and/or assistance. The IDP population also includes people in an IDP-like situation.
```

Comment:
Minor: there is a typo in the name of this field, but this name is used like that when it's called below.